homeassistant/components/webhook/__init__.py

```diff
@@ -86,10 +87,21 @@ def async_generate_id() -> str:
 
 @callback
 @bind_hass
-def async_generate_url(hass: HomeAssistant, webhook_id: str) -> str:
+def async_generate_url(
+    hass: HomeAssistant,
+    webhook_id: str,
+    prefer_external: bool = True,
+    allow_cloud: bool = False,
+    allow_ip: bool | None = None,
```

Comment:
This method is not meant for use with Home Assistant Cloud.

The Cloud integration has its own helpers for this (and also has different handling).

Reply:
fixed it to `False`